torch/csrc/api/include/torch/nn/functional/linear.h

```diff
@@ -12,6 +12,12 @@ inline Tensor bilinear(const Tensor& input1, const Tensor& input2, const Tensor&
 
 // ============================================================================
 
+inline Tensor bias(const Tensor& input, const Tensor& bias) {
+  return torch::bias(input, bias);
```

Comment:
Very nice, you resisted the temptation to just call add here :)

Reply:
it's not too late. i could even assert that it has the same behavior as calling the bias operator.